qcutilities.py

```diff
@@ -0,0 +1,38 @@
+from gempython.tools.vfat_user_functions_uhal import *
+
+import numpy as np
+import root_numpy as rp
+import ROOT as r
+import sys
+
+# rootTree - name of a TTree that has values that should have been written
+# dict_Names - dictionary where key names are the branch names in rootTree and the values are the register names they correspond too
+# device - optohybrid  the vfats belong to that you want to check
+# gtx - link of this optohybrid
```

Comment:
You can write this information as a [`docstring`](https://www.python.org/dev/peps/pep-0257/) so that it's a property of the function

Reply:
A la:
```python
def readBackCheck(rootTree, dict_Names, device, gtx):
"""<short one line description of method>

rootTree - name of a TTree that has values that should have been written
dict_Names - dictionary where key names are the branch names in rootTree and the values are the register names they correspond too
device - optohybrid  the vfats belong to that you want to check
gtx - link of this optohybrid
"""
...
```

Reply:
Ah okay, thanks for the example.  I will do this.